nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,4 @@
+NelmioApiDocBundle:
+    # Requires the Asset component and the Twig bundle
+    resource: "@NelmioApiDocBundle/Resources/config/routing/swaggerui.xml"
+    prefix:   /api/doc
```

Comment:
should this include a commented out definition for exposing the swaggerdoc json as well? (it should not be on by default, but having it in here seems useful to me)

    path: /api/doc.json
    methods: GET
    defaults: { _controller: nelmio_api_doc.controller.swagger }

Reply:
Sure, added :)